internal/provisioner/kops_provisioner.go

```diff
@@ -479,6 +481,7 @@ func (provisioner *KopsProvisioner) CreateClusterInstallation(cluster *model.Clu
 			},
 		},
 		Spec: mmv1alpha1.ClusterInstallationSpec{
+			Replicas:               1,
```

Comment:
I am guessing you didn't mean to change this?

Reply:
yes, since we don't have a license, it should be one.
As soon we introduce the license we can make a check here if there is a license we make replicas = 2 otherwise keep in 1